src/views/confirmation.njk

```diff
@@ -15,6 +15,12 @@
         })
       }}
 
+      <p class="govuk-body-l">
+        We’ve sent a confirmation email to
```

Comment:
What if we cant send an email to the given address?

Reply:
Absolutely valid comment. I thought about it as well and after conversation with Bart we will render error message when something failed. So there will be either email and text highlighting email sent or error message. In the future we might improve it.